projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -409,30 +409,23 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         if (index < 0 || index > (this.isRemote ? this.totalItemCount : this.igxForOf.length)) {
             return;
         }
-        // this.state.startIndex = index;
         const containerSize = parseInt(this.igxForContainerSize, 10);
         const scr = this.igxForScrollOrientation === 'horizontal' ?
-        this.hScroll.scrollLeft :
-        this.vh.instance.elementRef.nativeElement.scrollTop;
+            this.hScroll.scrollLeft : this.vh.instance.elementRef.nativeElement.scrollTop;
         const isPrevItem = index < this.state.startIndex || scr >  this.sizesCache[index];
+        let nextScroll = isPrevItem ? this.sizesCache[index] : this.sizesCache[index + 1] - containerSize;
```

Comment:
```typescript
this.sizesCache[index + 1]
```

To reviewers - this is safe because the sizesCache always contains 1 more item than the igxForOf collection (the last item right/bottom edge position)